assets/tests/hooks/useFetchRoutes.test.ts

```diff
@@ -13,23 +12,18 @@ jest.mock("../../src/api", () => ({
 describe("useFetchRoutes", () => {
```

Comment:
Testing that this returns a list of Routes seems like it could be useful.

Reply:
Unfortunately, that requires figuring out how to trigger effects when resolving promises without `act` acting up. I investigated this, but couldn't quite figure it out.

Perhaps I could test it with the `mockState` helper from `useFetchTimepoints.test.ts`?

Reply:
I don't know if it would work, but one idea is to return a mock result that wasn't a promise, but rather just an object that included a `then` function that returned a value.

Reply:
That worked! 🎊